MariaDB/server
# MDEV-34146: Remove duplicate #DEBHELPER# from MariaDB server postrm

- [x] *The Jira issue number for this PR is: MDEV-34146*


## Description
Moving to use Debian systemd install and uninstall scripts caused duplicated DEBHELP to server postrm script. Commit removes unneeded and makes postrm work better and pass lintian tests

## Release Notes
No notes are needed

## How can this PR be tested?
Build DEB packages with `debian/deb-autobake.sh` and run `lintian --display-info --pedantic  --allow-root ../*.changes` to check if `E: mariadb-server: duplicate-updaterc.d-calls-in-postrm mariadb [postrm:86]` is not there PR works

or see Salsa-CI run's lintian:
https://salsa.debian.org/illuusio/mariadb-server/-/commit/ce8ff3539d17a661ec67a347b4180abe808c11ff

## Basing the PR against the correct MariaDB version

## PR quality check
